### PR TITLE
Classic control benchmark

### PR DESCRIPTION
**Pull request by dekuenstle**
Changes:

* Predefine reward thresholds of convergence criteria for the particular environment
* Use max episode timesteps predefined by the particular environment
* Use mean of episode rewards as run performance
* Log mean performance every 10th episode
* Command line interface for running default configuration
* Stop at maximum episodes OR maximum timesteps

### CODE REVIEW

**Comment by KEggensperger** [comment]
Are you still working on this? Just ping me when this is ready to be reviewed.

**Comment by dekuenstle** [comment]
> Are you still working on this? Just ping me when this is ready to be reviewed.
@KEggensperger 
I just resolved an issue with installation and added a useful feature. Work on this should be done for now.

File: hpolib/benchmarks/rl/classic_control.py [hunk 1]
```diff
@@ -1,34 +1,48 @@
...
-from tensorforce.contrib.openai_gym import OpenAIGym
-from tensorforce.execution import Runner
+try:
+    from tensorforce.agents import PPOAgent
+    from tensorforce.contrib.openai_gym import OpenAIGym
+    from tensorforce.execution import Runner
+except ImportError:
+    raise ImportError(
+        "Missing tensorforce dependency. "
+        "Install extra 'rl': pip install hpolib2[rl].")
```
Comment (KEggensperger) [inline hpolib/benchmarks/rl/classic_control.py:20]: Add a version string here

File: hpolib/benchmarks/rl/classic_control.py [hunk 2]
```diff
@@ -196,3 +250,20 @@ def get_configuration_space():
         cs.add_hyperparameter(CS.UniformFloatHyperparameter(
             "entropy_regularization", lower=0, default_value=0.01, upper=1))
         return cs
+
+
+# Allow simple runs via the command line for testing and exploring the benchmark
+if __name__ == '__main__':
```
Comment (KEggensperger) [inline hpolib/benchmarks/rl/classic_control.py]: Could you please remove this or move this example to the example directory?

**Review by KEggensperger — CHANGES_REQUESTED** [review]
This looks great! Before merging, I have 3 really minor requests:

- Could you please add a version string _0.2_ (as in https://github.com/automl/HPOlib2/blob/development/hpolib/benchmarks/ml/autosklearn_benchmark.py#L25) 
- Add your changelog (the text of this PR) as a comment in the code (e.g. as a _note_ in the meta-dictionary) so we know why this has been changed
- Remove the command line interface (no other benchmark has this). If you think the code is particular useful, feel free to move it to the example directory

I will then take care of this benchmark and see whether I can rebase this branch. Thanks a lot!

**Comment by dekuenstle** [comment]
> This looks great! Before merging, I have 3 really minor requests:
> 
> * Could you please add a version string _0.2_ (as in https://github.com/automl/HPOlib2/blob/development/hpolib/benchmarks/ml/autosklearn_benchmark.py#L25)
> * Add your changelog (the text of this PR) as a comment in the code (e.g. as a _note_ in the meta-dictionary) so we know why this has been changed
> * Remove the command line interface (no other benchmark has this). If you think the code is particular useful, feel free to move it to the example directory
> 
> I will then take care of this benchmark and see whether I can rebase this branch. Thanks a lot!

Thank you for the review! I changed according to your suggestions.

**Review by KEggensperger — APPROVED** [review]

**Pull request merged**